Makefile

```diff
@@ -58,21 +58,25 @@ go_lint: go_deps
 go_test: go_deps
 	cd $(WPTD_GO_PATH); go test -v ./...
 
-bq_proto: $(PROTOS)
+bq_proto: submodules $(PROTOS)
 	mkdir -p $(PB_BQ_OUT_DIR)
 	protoc -I$(PB_LIB_DIR) -I$(PB_BQ_LIB_DIR) -I$(PB_LOCAL_LIB_DIR) \
 		--bq-schema_out=$(PB_BQ_OUT_DIR) $(PROTOS)
 
-go_proto: $(PROTOS)
+go_proto: submodules $(PROTOS)
 	mkdir -p $(PB_GO_OUT_DIR)
 	protoc -I$(PB_LIB_DIR) -I$(PB_BQ_LIB_DIR) -I$(PB_LOCAL_LIB_DIR) \
 		--go_out=$(PB_GO_PKG_MAP):$(PB_GO_OUT_DIR) $(PROTOS)
 
-py_proto: $(PROTOS)
+py_proto: submodules $(PROTOS)
 	mkdir -p $(PB_PY_OUT_DIR)
 	protoc -I$(PB_LIB_DIR) -I$(PB_BQ_LIB_DIR) -I$(PB_LOCAL_LIB_DIR) \
 		--python_out=$(PB_PY_OUT_DIR) $(PROTOS)
 
+# Ensure submodule have been pulled.
+submodules:
+	cd $(WPTD_PATH); git submodule update --init
```

Comment:
Just a nit: we probably don't need `cd $(WPTD_PATH)` as we are at this directory already.